docs/source/openshift/openshift-client-binary.adoc

```diff
@@ -27,12 +27,12 @@ export PATH="/Users/john/.minishift/cache/oc/v1.5.0:$PATH"
 ----
 
 [[minishift-oc-context]]
-== Minishift CLI Profile
+== {project} CLI Profile
 
-As a part of the `minishift start` command, a Minishift https://docs.openshift.org/latest/cli_reference/manage_cli_profiles.html[CLI profile] is also created.
+As a part of the `minishift start` command, a {project} https://docs.openshift.org/latest/cli_reference/manage_cli_profiles.html[CLI profile] is also created.
```

Comment:
I did not a thorough review, but here is another case where just replacing Minishift w/ CDK is probably not quite right

Reply:
Thanks, that actually gives me a lot of information about what does and does not apply here.

I'll look at this much more thoroughly and make some changes.

Reply:
Interestingly, CDK has this replaced with {cdk} in both the current docs branch and live documentation. Was that inaccurate to begin with?

Reply:
the profile is called _minishift_ regardless. So in case someone expect a literal CDK profile, he won't find one. I think the problem might be that in this context we should not use the project name at all, but use lowercase _minishift_ representing the actual name of the profile which gets created. Something like: "As a part of the `minishift start` command, a https://docs.openshift.org/latest/cli_reference/manage_cli_profiles.html[CLI profile] with the name _minishift_ is also created.